
show_config: true
debug: true

map_resolution: 0.05

map_h_inflaction: 0.1
map_v_inflaction: 0.1
z_not_inflate: 0.1

z_weight_cost: 1.2

goal_weight: 1.0

traj_dxy_max: 1
traj_pos_tol: 1
traj_yaw_tol: 0.1
traj_dz_max: 1
traj_vxy_m: 1
traj_vz_m: 1
traj_vxy_m_1: 1
traj_vz_m_1: 1
traj_wyaw_m: 1

world_frame: map

robot_base_frame: base_link